bin/oay/Cargo.toml

```diff
@@ -52,3 +64,5 @@ tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 url = "2.3.1"
 uuid = { version = "1", features = ["v4", "fast-rng"] }
+dav-server = { version = "0.5.5", optional = true }
+bytes = { version = "1.4.0", optional = true }
```

Comment:
Be careful with this line break.

Reply:
All files should end with a new file, so that we can avoid this in github:

![image](https://github.com/apache/incubator-opendal/assets/5351546/6afef790-be69-4304-a30d-ca7c2d53b58c)

But it's not a big issue, we can improve it later.